dds_web/development/db_init.py

```diff
@@ -78,309 +78,136 @@
-        #                      "A1BE27E07B9694061DADBD2B581E0AB0D8"),
-        #         salt="9A2694986A488E438DA998E73E93E9C4",
-        #         nonce="D55240DB506C2C22CF248F18",
-        #         passphrase="922d5b93f5455050e96b33a45f65a3e8c7d4f6198ed8473879c11e10711ed937"),
-        # Project(id="IMG005", title="Mitochondrial translation",
-        #         category="Imaging",
-        #         order_date="2020-11-11", delivery_date=None,
-        #         status="Ongoing", sensitive=False,
-        #         description="Distinct pre-initiation steps in human translation",
-        #         pi="", owner=users[3], facility=facilities[2], size=0,
-        #         size_enc=0, delivery_option="DC_DDS",
-        #         public_key=("092C894633C31C3EF9E90A3EF2400E4AC4D2ADE8BF"
-        #                     "7EFECAB889829253136B33"),
-        #         private_key=("83B7C905A0C7AA9B95456440CBC80956CB53CABF19AE76B01A"
-        #                      "A5F7324FFA74CBAC1350DDF760BFDBDF94CD6314D3F7418E37"
-        #                      "064954751F7320B9CE83E1DDCE1CB1412564C536E3221D07F3"
-        #                      "A1BE27E07B9694061DADBD2B581E0AB0D8"),
-        #         salt="9A2694986A488E438DA998E73E93E9C4",
-        #         nonce="D55240DB506C2C22CF248F18",
-        #         passphrase="922d5b93f5455050e96b33a45f65a3e8c7d4f6198ed8473879c11e10711ed937")
     ]
 
-    roles = [
-        Role(username=users[0].username, facility=False),
-        Role(username=users[1].username, facility=False),
-        Role(username=facilities[0].username, facility=True),
+    users = [
+        User(
+            public_id="public_user_id",
+            username="username",
+            password="$argon2id$v=19$m=102400,t=2,p=8$0jcemW3Ln+HTPUt/E3xtKQ$aZGqrrBBU5gq5TbWYwUWD62UiQUmTksbKOkmbMJzdhs",
+            role="researcher",
+            permissions="-gl--",
+            facility_id=None,
+        ),
+        User(
+            public_id="public_admin_id",
+            username="admin",
+            password="$argon2id$v=19$m=102400,t=2,p=8$0jcemW3Ln+HTPUt/E3xtKQ$aZGqrrBBU5gq5TbWYwUWD62UiQUmTksbKOkmbMJzdhs",
+            role="admin",
+            permissions="a-l--",
+            facility_id=None,
+        ),
+        User(
+            public_id="public_facility_admin_id",
+            username="facility_admin",
+            password="$argon2id$v=19$m=102400,t=2,p=8$0jcemW3Ln+HTPUt/E3xtKQ$aZGqrrBBU5gq5TbWYwUWD62UiQUmTksbKOkmbMJzdhs",
+            role="facility",
+            permissions="a-l--",
+            facility_id=facilities[0],
+        ),
+        User(
+            public_id="public_facility_id",
+            username="facility",
+            password="$argon2id$v=19$m=102400,t=2,p=8$0jcemW3Ln+HTPUt/E3xtKQ$aZGqrrBBU5gq5TbWYwUWD62UiQUmTksbKOkmbMJzdhs",
+            role="facility",
+            permissions="--lpr",
+            facility_id=facilities[0],
+        ),
     ]
 
+    # User(
```

Comment:
Can remove this ? not a important one, just feel like we never use these old codes later (even for ref)

Reply:
Yes, will remove it.